packages/slice-machine/src/features/navigation/UpdateInfo.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Button,
+  Card,
+  CardContent,
+  Text,
+  useMediaQuery,
+} from "@prismicio/editor-ui";
+import Link from "next/link";
+
+import { useUpdateAvailable } from "@/hooks/useUpdateAvailable";
+
+export function UpdateInfo() {
+  const isCollapsed = useMediaQuery({ max: "medium" });
+
+  const { sliceMachineUpdateAvailable, adapterUpdateAvailable } =
+    useUpdateAvailable();
+
+  if (
+    (!sliceMachineUpdateAvailable && !adapterUpdateAvailable) ||
+    isCollapsed
+  ) {
+    return null;
+  }
+
+  return (
+    <Card color="grey3" padding={16} sx={{ marginTop: 32 }}>
```

Comment:
asymmetrical component here, in this case it's even worse i'd say because we're pushing down this component 32px no matter where it's rendered - the parent should be responsible for this